plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/available-filters/checkout-and-place-order-button.md

```diff
@@ -191,16 +191,16 @@ registerCheckoutFilters( 'example-extension', {
 
 ### Description <!-- omit in toc -->
 
-The `placeOrderButtonLabel` filter allows change the label of the "Place order" button.
+The `placeOrderButtonLabel` filter allows change the label of the "Place order · <price/>" button.
 
 ### Parameters <!-- omit in toc -->
 
--   _defaultValue_ (type: `string`, default: `Place order`) - The label of the "Place order" button.
+-   _defaultValue_ (type: `string`, default: `Place order · <price/>`) - The label of the "Place order · <price/>" button.
```

Comment:
```suggestion
-   _defaultValue_ (type: `string`, default: `Place order · <price/>`) - The label of the "Place order · &lt;price/>" button.
```

Similar to my previous comment. We only need to escape the opening bracket in the plain text. The one, that's used within the inline code, is fine, see:

<img width="855" alt="Screenshot 2024-05-16 at 12 11 57" src="https://github.com/woocommerce/woocommerce/assets/3323310/defe1d37-6e25-4416-9a91-6d0b2b272f34">